### PR TITLE
A terse way to handle different HTTP methods

### PR DESCRIPTION
**Pull request by vector-of-bool**

### CODE REVIEW

File: node/mo/pages/api/links/index.ts [hunk 1]
```diff
@@ -1,32 +1,20 @@
...
-    const response = handleCase[method]
-    if (response) await response(req, res)
-    else res.status(400).json({ error: "No Response for This Request" })
-}
-
-export default handler
+export default createMethodHandler({
+  async get(req, resp) {
+    const { MoLink } = await connect();
+    resp.json(await MoLink.find().sort({ n: -1 }).limit(100));
```
Comment (ShaneHarvey) [inline node/mo/pages/api/links/index.ts:7]: Isn't this missing the error handling now?

Comment (vector-of-bool) [reply node/mo/pages/api/links/index.ts:7]: I believe that errors here will be handled by NextJS as HTTP 500 errors, but I may be wrong.

Comment (ShaneHarvey) [reply node/mo/pages/api/links/index.ts:7]: Could you confirm by making the sort invalid?

Comment (vector-of-bool) [reply node/mo/pages/api/links/index.ts:7]: Confirmed. It generates an HTTP 500 error report with a backtrace in the log and an error page in the UI.

**Review by ShaneHarvey — APPROVED** [review]

File: node/mo/pages/api/links/index.ts (hunk 1 above)
Comment (ShaneHarvey) [reply node/mo/pages/api/links/index.ts:7]: Thanks

**Pull request merged**